dcli/lib/src/process/process/process_in_isolate.dart

```diff
@@ -65,7 +70,7 @@ void _startIsolate(ProcessSettings processSettings, ProcessChannel channel) {
 
           /// The tell the sender that we got their data and
           /// sent it to stdin
-          sendMailbox.respond(Uint8List.fromList([ProcessChannel.RECEIVED]));
+          sendMailbox.put(Uint8List.fromList([ProcessChannel.RECEIVED]));
```

Comment:
Send message back to main thread with 'put(..)'

Reference Here:
https://github.com/dart-lang/native_synchronization/blob/fc4ee3f3b5cdd439ed4c18f2003468ceb24e6458/lib/mailbox.dart#L71